src/js/sdk.tsx

```diff
@@ -65,10 +69,10 @@ export function init(passedOptions: ReactNativeOptions): void {
     ];
 
     if (__DEV__) {
-      options.defaultIntegrations.push(new DebugSymbolicator());
+      passedOptions.defaultIntegrations.push(new DebugSymbolicator());
```

Comment:
@marandaneto  or @AbhiPrasad should we push those integrations to the defaultIntegration or the integration itself from options? (options no longer has a parameter called defaultIntegration.

Reply:
let's push this onto integrations option key and stay away from mutating default integrations.